byzcoin/bcadmin/test.sh

```diff
@@ -28,4 +33,80 @@ testCreateStoreRead(){
     testFail ./bcadmin add spawn:xxx
 }
 
+testAddDarc(){
+  runCoBG 1 2 3
+    runGrepSed "export BC=" "" ./"$APP" create --roster public.toml --interval .5s
```

Comment:
Something is wrong with formatting in this file, if you can find an auto-formatter for shell, please tell us about it, so we all get the benefit!